sycl/unittests/helpers/PiImage.hpp

```diff
@@ -206,10 +206,90 @@ class PiImage {
         MLinkOptions(LinkOptions), MManifest(std::move(Manifest)),
         MBinary(std::move(Binary)), MOffloadEntries(std::move(OffloadEntries)),
         MPropertySet(std::move(PropertySet)) {
+    initializeBinaryDesc(Version, Kind, Format);
+  }
+
+  /// Constructs a SYCL device image of the latest version.
+  PiImage(uint8_t Format, const std::string &DeviceTargetSpec,
+          const std::string &CompileOptions, const std::string &LinkOptions,
+          std::vector<unsigned char> Binary,
+          PiArray<PiOffloadEntry> OffloadEntries, PiPropertySet PropertySet)
+      : PiImage(PI_DEVICE_BINARY_VERSION, PI_DEVICE_BINARY_OFFLOAD_KIND_SYCL,
+                Format, DeviceTargetSpec, CompileOptions, LinkOptions, {},
+                std::move(Binary), std::move(OffloadEntries),
+                std::move(PropertySet)) {}
+
```

Comment:
Why can't we use default for this one? Please add a comment.

Reply:
This one was not added in this PR, it was here before: https://github.com/intel/llvm/pull/7070/files#diff-0e91a022ae2eb241758604e815c0f50613d9515ce8ef0ad5c865918f809cc91eL233-L245